core/management/commands/purge_essays.py

```diff
@@ -1,15 +0,0 @@
-from django.core.management.base import BaseCommand
```

Comment:
Do they load from a URL?  It seems like this command is for purging *all* essays, and it purges by URL.  I get the feeling it's for doing a bulk destroy followed by a bulk reload.

Maybe a command for purging a single essay by URL could be useful, but if we don't have a good way to load essays in bulk, this command could really hose somebody.